File: matchingStrings/matching_strings.py
import math
import os
import random
import re
import sys

stringsOne = [ 'aba', 'baba', 'aba', 'xzxb' ]
queriesOne = [ 'aba', 'xzxb', 'ab' ]


def matching_strings(strings, queries):
    answer = []
    for i in queries:
        counter = 0
        for j in strings:
            if [i] == [j]:
                counter+=1
        answer.append(counter)
    print(answer)
    return(answer)


matching_strings(stringsOne, queriesOne)
    
        

File: matchingStrings/matchingStrings.js
const stringsOne = [ 'aba', 'baba', 'aba', 'xzxb' ]
const queriesOne = [ 'aba', 'xzxb', 'ab' ]


function matchingStrings(strings, queries) {
    let answer = []
    let answerTwo = ['2', '1', '0']

    for(i=0; i<queries.length; i++){
        let counter = 0
        for(j=0; j<strings.length; j++){
            if (queries[i] == strings[j]){
                counter += 1
            }
        }
        answer.push(counter)
    }
    console.log(answer)
    return answer

}

matchingStrings(stringsOne, queriesOne)